Misc/NEWS.d/next/Library/2025-04-05-02-22-49.gh-issue-132106.XMjhQJ.rst

```diff
@@ -0,0 +1,2 @@
+:class:`logging.handlers.QueueListener` now implements the context

```

Comment:
Can you also create a separate blurb but with the same issue that indicates that a RuntimeError is now raised if a listener is started more than once?

Reply:
Yep. Will split this out.